driver/src/test/java/org/neo4j/driver/v1/integration/CausalClusteringIT.java

```diff
@@ -85,6 +86,12 @@
     @Rule
     public final ClusterRule clusterRule = new ClusterRule();
 
+    @AfterClass
+    public static void stopSharedCluster()
+    {
+        ClusterRule.stopSharedCluster();
```

Comment:
Why we want to immediately shop the cluster after this test class? Should we actually want to keep the cluster running until the whole tests finishes?

If this is the only place that requires a cluster, then maybe we should change the ClusterRule to be a class role?

Reply:
Cluster takes a lot of memory and might influence other tests. There exists one more test that uses this rule. I can make it a `@ClassRule` and add cleaning of db in some `@After` method instead.